vendor/vendor.json

```diff
@@ -420,23 +461,46 @@
 			"path": "github.com/aws/aws-sdk-go-v2/service/ec2/ec2iface",
 			"revision": "d52522b5f4b95591ff6528d7c54923951aadf099",
 			"revisionTime": "2018-09-27T22:51:20Z",
-			"version": "v2.0.0-preview.5",
+			"tree": true,
+			"version": "v2.0.0-preview.9",
 			"versionExact": "v2.0.0-preview.5"
 		},
+		{
+			"checksumSHA1": "qUIf77TBORA1MZoNFNdiR5kFRjI=",
+			"path": "github.com/aws/aws-sdk-go-v2/service/elasticloadbalancingv2",
+			"revision": "db223f6d5a456d1c9dfaeb7ef865cc63e03ec2ce",
+			"revisionTime": "2019-06-11T23:54:58Z"
+		},
+		{
+			"checksumSHA1": "kCyrtNPVHzoajcxIYke+pHqWkR0=",
+			"path": "github.com/aws/aws-sdk-go-v2/service/elasticloadbalancingv2/elasticloadbalancingv2iface",
+			"revision": "db223f6d5a456d1c9dfaeb7ef865cc63e03ec2ce",
+			"revisionTime": "2019-06-11T23:54:58Z",
+			"tree": true,
+			"version": "v2.0.0-preview.9"
+		},
+		{
+			"path": "github.com/aws/aws-sdk-go-v2/service/elasticloadbalancingv2iface",
```

Comment:
Hmm I can't find this path `github.com/aws/aws-sdk-go-v2/service/elasticloadbalancingv2iface` for elasticloadbalancingv2iface in https://github.com/aws/aws-sdk-go-v2/tree/master/service